lib/view_model/active_record/nested_controller_base.rb

```diff
@@ -7,6 +7,40 @@
 module ViewModel::ActiveRecord::NestedControllerBase
   extend ActiveSupport::Concern
 
+  class ParentProxyModel < ViewModel
+    # Prevent this from appearing in hooks
+    self.synthetic = true
+
+    attr_reader :parent, :association_data, :changed_children
+
+    def initialize(parent, association_data, changed_children)
+      @parent = parent
+      @association_data = association_data
+      @changed_children = changed_children
+    end
+
+    def serialize(json, serialize_context:)
+      ViewModel::Callbacks.wrap_serialize(parent, context: serialize_context) do
+        child_context = parent.context_for_child(association_data.association_name, context: serialize_context)
+
+        json.id parent.id
+        json.update do
```

Comment:
perhaps use `ViewModel::BULK_UPDATE_ATTRIBUTE` here to be clearer that we're emitting part of the `_bulk_update` structure.